Formula/codeclimate.rb

```diff
@@ -10,7 +10,7 @@ class Codeclimate < Formula
   def install
     # Alter PATH to ensure `docker' is available
     system "env", \
-      "PATH=/usr/local/bin:#{ENV["PATH"]}", \
+      "PATH=#{HOMEBREW_PREFIX}/bin:#{ENV["PATH"]}", \
```

Comment:
As far as I remember brew always define this constant (https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/config.rb#L33-L34).